examples/openshift/README.md

```diff
@@ -31,9 +31,9 @@ Deploy the charts:
 ```shell
 helm upgrade --install --namespace spire-server spire charts/spire \
 --values examples/production/values.yaml \
---values examples/production/example-your-values.yaml \
---values examples/tornjak/values.yaml \
 --values examples/openshift/openshift-values.yaml \
+--values examples/tornjak/values.yaml \
+--values examples/production/example-your-values.yaml \
```

Comment:
Reorder a bit, especially so that the users values always comes last, so they can override anything they need to.